fix: build change to npm having issue with pnpm

varun2948 opened the pull request.

spwoodcock reviewed (commented):
What are the issues? I'm sure its solvable 😁

Definitely prefer pnpm

varun2948 commented:
oh yes sam, was having issue with the pnpm cache and changed it to npm to remove the cache and switched back again to pnpm . It was weird issue so have to do that but switched back so its good for pnpm
